support.md

```diff
@@ -10,7 +10,8 @@ title: Get Support
 * Discuss on the [User mailing list](https://groups.google.com/forum/#!forum/bazel-discuss).
 * Report bugs or feature requests in our [GitHub issue tracker](https://github.com/bazelbuild/bazel/issues).
 * Find other Bazel contributors on [IRC](http://webchat.freenode.net)
-(irc.freenode.net#bazel).
+(irc.freenode.net#bazel) or [Slack](https://bazelbuild.slack.com) (get an
```

Comment:
Here we can list irc, but with the same comments as the contibuting file

Reply:
done.